src/components/Navbar.tsx

```diff
@@ -1,15 +1,144 @@
+    backgroundColor: '#FFFFFF',
+    '&:hover': {
+      backgroundColor: '#EEEEEE',
+    },
+    marginRight: theme.spacing(2),
+    marginLeft: 0,
+    width: '100%',
+    [theme.breakpoints.up('sm')]: {
+      marginLeft: theme.spacing(3),
+      width: 'auto',
+    },
+  },
+  searchIcon: {
+    padding: theme.spacing(0, 2),
+    height: '100%',
+    position: 'absolute',
+    pointerEvents: 'none',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    zIndex: 1200,
+  },
+  inputRoot: {
+    color: 'inherit',
+  },
+  inputFocused: {
+    backgroundColor: '#EEEEEE',
+  },
+  inputInput: {
+    padding: theme.spacing(1, 1, 1, 0),
+    // vertical padding + font size from searchIcon
+    paddingLeft: `calc(1em + ${theme.spacing(4)}px)`,
+    transition: theme.transitions.create('width'),
+    width: '100%',
+    [theme.breakpoints.up('md')]: {
+      width: '20ch',
+    },
+  },
+}));
+
+function Navbar(props: any) {
+  // eslint-disable-next-line no-shadow
+  const { loggedIn, submitSearch, changeFilters }: { loggedIn: boolean, submitSearch: Function, changeFilters: Function } = props;
+  const classes = useStyles();
+  const currFilters: Filters = {
+    query: '',
+    letterCodes: [],
+    numberRange: [MIN_COURSE_CODE, MAX_COURSE_CODE],
+  };
+
+  const handleQueryChange = (event: React.ChangeEvent<{ value: unknown }>) => {
+    const { value } = event.target as HTMLSelectElement;
+    if (value.trim() === '' || value.trim() === '') {
+      return;
+    }
+    currFilters.query = value;
+    changeFilters(currFilters);
+  };
 
-function Navbar() {
   return (
-    <div>
-      <Link to="/">Brandon</Link>
-      <Link to="/Browse">Browse</Link>
-      <Link to="/Sam">Sam</Link>
-      <Link to="/Guanting">Guanting</Link>
-    </div>
+    <AppBar className={classes.navBar} position="static" color="transparent">
+      <Toolbar>
+        <Typography className={classes.title} variant="h5">
+          <Link href="#">
+            CourseFlo
+          </Link>
+        </Typography>
+        {/* <div> */}
+        {/*  <Grid container spacing={1} alignItems="flex-end"> */}
+        {/*    <Grid item> */}
+        {/*      <SearchIcon /> */}
+        {/*    </Grid> */}
+        {/*    <Grid item> */}
+        {/*      <Input */}
+        {/*        placeholder="Course Search…" */}
+        {/*      /> */}
+        {/*    </Grid> */}
+        {/*  </Grid> */}
+        {/* </div> */}
+        <div className={classes.search}>
+          <div className={classes.searchIcon}>
+            <SearchIcon />
+          </div>
+          <InputBase
+            placeholder="Course Search…"
+            classes={{
+              root: classes.inputRoot,
+              input: classes.inputInput,
+              focused: classes.inputFocused,
+            }}
+            inputProps={{ 'aria-label': 'search' }}
+            onChange={handleQueryChange}
+            onSubmit={submitSearch(currFilters)}
```

Comment:
I just realized: Can we make this into a redirect, where the submit search is called first like so:
```
const redirectToBrowse = () => {
  submitSearch(currFilters);
  ///// redirect somehow??
  redirect();
}
```

```suggestion
            onSubmit={redirectToBrowse}
```